### PR TITLE
Update MSTR.md

### PR DESCRIPTION
**Pull request by Libby-Monarch**
Added the following:
- 90 minutes as an option for time limits
- End of Match procedure (w/examples) for 90 minute timed rounds

### CODE REVIEW

File: MSTR.md [hunk 1]
```diff
@@ -117,11 +117,17 @@ _Example: Alice is ranked first in the last round of swiss. This rule prevents h
...
 
-**2.5B.** In Multiplayer tournaments, it is strongly recommended that single elimination matches be untimed. Any deviation must be communicated to the players before tournament begins and any tiebreaker method should be based solely on information pertaining to the current match.
+**2.5B.** In Multiplayer tournaments where 90 minutes are used as the round time limit, when time in the round is reached the match ends immediately. If a spell or ability is in the process of resolving, it will finish doing so completely. State-based actions are then checked. If no winner has been determined at that time, the match is considered a draw.
```
Comment (fbatista) [inline MSTR.md:124]: Interesting.

I've seen 90-minute rounds that end at the end of the current turn, but this take is intriguing, and I had actually thought about something similar to this, but with an opportunity to still punish slow play like in 75 minutes:
- 90 minute rounds; 75 minutes + 4 extra turns, but the 4 extra turns must be completed within 15 minutes, and if the 15 minutes are depleted, the game ends immediately.

Another option that I had thought about was 90 minute rounds, ending on the end step. Introduce a different additional remedy for slow play: the player performing slow play automatically passes priority on everything until the beginning of the next upkeep.

WDYT? Can you share some insight regarding the reasoning behind the 90 minutes hard-cap solution?

**Review by fbatista — APPROVED** [review]
This solution can work, but I'm a bit worried that 90 minutes hard-cap can potentially open more slow play abuse. In the declare attackers example you proposed, the abuse potential is extremely glaring! The defending player can just "think" on how to declare blockers etc. sure, if players call the judge, they can detect the stalling, however, what if they don't?

File: MSTR.md (hunk 1 above)
Comment (Libby-Monarch) [reply MSTR.md:124]: You're right. I was going to include some verbiage warning judges to be extra vigilant for slow play/stalling, and based on your comment I will.

Comment (Libby-Monarch) [reply MSTR.md:124]: These suggestions are interesting. Let me go consider :)

**Pull request closed without merging**